stock_barcodes_zpl2/wizard/print_record_label.py

```diff
@@ -0,0 +1,131 @@
+# Copyright 2020 ForgeFlow S.L. (<http://www.forgeflow.com>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+_supported_models = [
+    "stock.picking",
+    "stock.inventory",
+    "product.product",
+    "product.template",
+]
+
+
+class PrintRecordLabel(models.TransientModel):
+    _inherit = "wizard.print.record.label"
+
+    model = fields.Char(related="active_model_id.model", string="Model Name")
```

Comment:
Mmmh, IMHO, those fields are too abstract to be in a stock module, no ?

Reply:
Great. Maybe you can add a Selection field for supported documents there too (for the show_lines)?

Reply:
Where? How? I don't get you now :S